### ISSUE TITLE
Error when connect with two phones on same TV

### ISSUE THREAD

**Issue by Coger541** [odyshewroman/AndroidTVRemoteControl#31]
Hello @odyshewroman 

How are you bro!

Have you ever test this project with more than 1 phones on same smart tv. I did the same thing, I first try to connect 2 iPhone with same smart TV (Philips) but when I did this and try to send the code to smart tv (code shown on tv) then the project crashed.

Any idea regarding this issue ?

**Comment by odyshewroman** [comment]
Yo!
It's interesting. Yes, you can't have multiple connection in one Pairing\Remote manager, cause we have persistent connection. (same like you can't write into socket from different sources or to different destinations)
But it's shouldn't lead to crash. Now I temporary live in other region in deep Siberia, I there isn't any Smart TV, so I can't test it, but you will help me if you debug this case and send me the line, stack trace and crash message.

**Comment by Coger541** [comment]
Yes sure, I will send the logs soon. Currently I am busy in another project. Once I finished that I will try to send you logs

Also when you will again start to work on this project then please try to fix this issue

Thanks man for fast reply
